### ISSUE TITLE
notification not sent

### ISSUE THREAD

**Issue by ilkosta** [massdest/redmine_telegram_email#4]
Hi,

I'm trying the plugin by a docker-redmine but without success.
Trying the plugin I got the log:

```
TELEGRAM EMAIL TOKEN EMPTY, PLEASE SET IT IN PLUGIN SETTINGS
telegram_url https://api.telegram.org/bot/sendMessage
SPEAK TO TELEGRAM ...
```

But in the settings the token is not empty. Why?

**Comment by massdest** [comment]
Hello, it may not work with newer redmine versions, show me screenshot of your plugin settings page

**Comment by massdest** [comment]
ok, I made fix for 3.4 , check it now please

**Comment by massdest** [comment]
Thank you, excellent.
